### DISCUSSION TITLE
Embark on Your Next Career Chapter: Opportunities Awaits | Now Hiring

### DISCUSSION THREAD

**Discussion by TheTopTechRecruiter** [TheTopTechRecruiter/TheTopTechRecruiter#1] [General]
Greetings, talented Software Developers and Engineers! If you're on the lookout for new and exciting career opportunities to elevate your skills and take your professional journey to the next level, you've landed in the right space. This forum is dedicated to connecting innovative software developers to new career opportunities in a 100% remote setting.

In today's ever-evolving tech landscape, your expertise is in high demand, and @TheTopTechRecruiter is actively seeking dynamic individuals like you to contribute to their success. Whether you're a seasoned professional exploring new challenges or a recent graduate eager to kickstart your career, this platform is designed to be your gateway to a world of possibilities.

Join our community of like-minded developers who have navigated diverse career paths and gained valuable insights along the way. Discover opportunities that align with your passions, leverage your unique skill set, and connect with forward-thinking companies looking to build exceptional teams.

Your next career adventure awaits – let's explore it together. Dive into this vibrant discussion, share your experiences, and be inspired as you embark on the journey to your dream job. Welcome to a world of exciting possibilities!

### **Currently Hiring:**

**Principal Developer:**
- React/Typescript

**Senior Developers:**
- Web (React / Angular)
- .Net
- Java
- Full Stack (Java w/ Angular, Java w/ vue.js)
- QA Automation 

Let Connect!
https://www.linkedin.com/in/cameronmabb/

**Comment by TheTopTechRecruiter** [comment]
If anyone is interested in discussing potential roles feel free to reach out!

**Comment by Eagle9461** [comment]
Hi,
Thanks for your posting.

I'm very intersted in your post, and I think I have all skills you required.

https://drive.google.com/file/d/1NPrjknLNrr-qAGrnvsyDB5i9Dt8XkHVZ/view?usp=sharing

My resume link is here.

**Reply by TheTopTechRecruiter** [reply]
Hi Eagle9461,

Please send a pdf of your resume to me at cameron.mabb@teamsparq.com

**Comment by TheTopTechRecruiter** [comment]
3/18/2024
Now hiring: Principal Azure Devops Engineer
                    Senior Azure Engineer

**Comment by TheTopTechRecruiter** [comment]
3/28/2024
Now Hiring: 
Full Stack - .Net/Angular
Devops - Azure
AWS Engineer

**Comment by TheTopTechRecruiter** [comment]
- 👋 Hi, I’m Cameron @TheTopTechRecruiter
- 👀 Hiring Software Developers for Full Time / 100% remote jobs
- Open to candidates in the United States and Mexico
- 🌱 Hiring for Practice Leader - Cloud, US 100% remote 
- 🌱 Hiring for SAP Basis Consultants in Mexico
- 🌱 Hiring for Mobile Devs w/ React Native experience in Miami, Seattle, Minneapolis, Dallas or Atlanta 
- 📫 How to reach me (https://www.linkedin.com/in/cameronmabb)
- cameron.mabb@teamsparq.com
